frontend/src/metabase/modes/components/drill/SortAction.jsx

```diff
@@ -11,6 +15,7 @@ export default ({ question, clicked }) => {
     !clicked ||
     !clicked.column ||
     clicked.value !== undefined ||
+    _.any(INVALID_TYPES, type => isa(clicked.column.base_type, type)) ||
```

Comment:
This probably isn't how I'd structure this check ex nihlo, but there's an established patterns of having an array of types to check against using this pattern in `DistributionDrill.jsx` and I think it makes sense to keep it consistent.